README.md

```diff
@@ -433,8 +433,6 @@ Rule selection:
           Comma-separated list of rule codes to disable
       --extend-select <RULE_CODE>
           Like --select, but adds additional rule codes on top of the selected ones
-      --extend-ignore <RULE_CODE>
-          Like --ignore, but adds additional rule codes on top of the ignored ones
```

Comment:
I do _kind_ of wonder if we should just leave this visible in the CLI to avoid confusion (due to the asymmetry with `--select`). What do you think? Bad idea?

Reply:
If we leave it I think we should add `DEPRECATED` to the description ... however I don't think that would really reduce the confusion since we'd have to explain why it's deprecated ... which would probably be too verbose. I consider `--extend-ignore` to effectively be an alias for `--ignore` ... and we currently don't document e.g. that `ruff clean` has `ruff --clean` as an alias in the CLI help.

Reply:
I think I prefer to hide it. I have updated the message to mention that it's deprecated since it still shows up in the clap shell completion.